doc/nrf/dev_model_and_contributions/adding_code.rst

```diff
@@ -162,7 +162,7 @@ This is demonstrated by the following code, that would be placed somewhere in yo
        - name: nrf
          repo-path: sdk-nrf
          remote: ncs
-         revision: 2.6.99-cs1
+         revision: 2.6.1
```

Comment:
```suggestion
         revision: 2.6.99
```

Reply:
Not in this particular case, here we show how to check out the latest stable release. Revision is changed to 2.6.99 in conf.py.